Automattic/sensei
# Add email preview link to the edit screen

Resolves #6465

## Proposed Changes
* Replace the default preview button with a custom preview link in the email edit screen.
* The implementation is based on the [`PostPreviewButton` component](https://github.com/WordPress/gutenberg/blob/f101c236d12a09cb31e166d15df59fb7458104d1/packages/editor/src/components/post-preview-button/index.js#L109) in Gutenberg. The `PostPreviewButton` component uses a somewhat complicated loading screen which is simplified in the proposed implementation.

## Testing Instructions


1. Enable the feature flag: `add_filter( 'sensei_feature_flag_email_customization', '__return_true' );`
2. Go to Sensei LMS -> Settings -> Emails.
3. Edit an email post.
4. In the edit screen, click the "Preview" button at the top.
5. Make sure you are redirected to a new tab with a preview of the email.
6. Go back to the edit screen and make some changes to the content of the post without saving it.
7. Click the "Preview" button and check if those changes are reflected in the preview.

## Pre-Merge Checklist
